src/vm/arm/ehhelpers.S

```diff
@@ -99,6 +99,7 @@ OFFSET_OF_FRAME=(4 + SIZEOF__GSCookie)
         NESTED_ENTRY CallEHFunclet, _TEXT, NoHandler
 
         PROLOG_PUSH  "{r4-r11, lr}"
+        PROLOG_STACK_SAVE r7
         alloc_stack  4
 
```

Comment:
@rahku @janvorli When I removed .pad instruction from ``free_stack``, I got the following error:
```
In try
In try 2, 1st throw
In 1st catch, 2nd throw
terminate called after throwing an instance of 'PAL_SEHException'
```